src/main/java/net/atos/zac/policy/PolicyService.java

```diff
@@ -95,6 +95,9 @@ public ZaakRechten readZaakRechten(final Zaak zaak, final ZaakType zaaktype) {
         return readZaakRechten(zaak, zaaktype, null);
     }
 
+    /**
+     * !!! user parameter is to be removed when we switch to paging
```

Comment:
Not a big deal but I'm not in favour of added such TODO's to the code in general. Can we not add a `@Deprecated` annotation to this parameter?

Reply:
Or add a second method without the user param and make this one deprecated?

Reply:
We need FE and BE to use the new implementation. So this seemed like the fastest way to do that as I wanted to keep the endpoints in their current form: `GET /zaken/{type}`

Deprecation IMHO require more steps:
1. new method on a new endpoint + deprecation of the old
2. change of FE
3. change on BE side + introduction of the same logic on the old endpoint
4. change on FE side
5. cleanup on BE

Currently it's only 3 steps:
1. new behavior on BE
2. change on FE
3. cleanup

Reply:
Used `@Deprecated` to replace the `!!!` comment